_dev/src/components/block/dispute.vue

```diff
@@ -17,24 +17,27 @@
  * International Registered Trademark & Property of PrestaShop SA
  *-->
 <template>
-  <div class="card">
-    <div class="card-block text-center py-4 px-5">
-      <h1>0 {{ $t('block.dispute.pendingDispute') }}</h1>
-      <a href="https://www.paypal.com/disputes/" target="_blank">
-        {{ $t('block.dispute.goToDispute') }} <i class="material-icons">arrow_right_alt</i>
-      </a>
-    </div>
-  </div>
+  <b-card no-body>
+    <b-card-body class="py-2">
+      <b-row>
+        <b-col>
+          <b-button variant="text" size="lg">
+            <i class="material-icons">school</i>
+            0 {{ $t('block.dispute.pendingDispute') }}
```

Comment:
`0` should replace a placeholder. Following the doc (https://kazupon.github.io/vue-i18n/guide/formatting.html#named-formatting), we should see something like

```suggestion
            {{ $t('block.dispute.pendingDispute', { disputeCount: 0}) }}
```

With block.dispute.pendingDispute updated as `{disputeCount} pending dispute(s)`